crates/holochain/src/core/ribosome/host_fn/create_x25519_keypair.rs

```diff
@@ -2,24 +2,23 @@ use crate::core::ribosome::error::RibosomeResult;
 use crate::core::ribosome::CallContext;
 use crate::core::ribosome::RibosomeT;
 use holochain_keystore::keystore_actor::KeystoreSenderExt;
-use holochain_zome_types::CreateX25519KeypairInput;
-use holochain_zome_types::CreateX25519KeypairOutput;
 use std::sync::Arc;
+use holochain_zome_types::X25519PubKey;
 
 pub fn create_x25519_keypair(
     _ribosome: Arc<impl RibosomeT>,
     call_context: Arc<CallContext>,
-    _input: CreateX25519KeypairInput,
-) -> RibosomeResult<CreateX25519KeypairOutput> {
-    Ok(CreateX25519KeypairOutput::new(
+    _input: (),
```

Comment:
see above

Reply:
yes this will never have an input because it creates a new keypair from pure entropy